kopf/reactor/queueing.py

```diff
@@ -183,13 +183,8 @@ def create_tasks(
     tasks = []
 
     # Monitor the peers, unless explicitly disabled.
-    if not standalone:
-        ourselves = Peer(
-            id=detect_own_id(),
-            priority=priority,
-            peering=peering,
-            namespace=namespace,
-        )
+    ourselves: Optional[Peer] = PeerFactory.create_peer(standalone, peering, id=detect_own_id(), priority=priority, namespace=namespace)
```

Comment:
I didn't know it is syntactically allowed to do so in the code flow. It is nice to learn something new :-)

Reply:
it's introduced in python3.7, quite new :D

Reply:
Oh! Good that I've started this project with `>=3.7` requirement, not `>=3.6`! :-) (Also, some 3.7-only ISO datetime functions are used.) — Absence of legacy is always good.